src/main/java/com/onlinebookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package com.onlinebookstore.dto.user;
+
+import com.onlinebookstore.validation.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserLoginRequestDto {
+    @NotNull
+    @Size(min = 4, max = 50)
+    @Email
+    private String email;
```

Comment:
```suggestion
    @Email
    private String email;
```
check the length and if null in email validator

Reply:
Thank you!